src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -21,27 +21,238 @@ class er;
+ * (bilinear).
+ *
+ * \image html ProjectionOntoPlane.png "Controlling the degree of projection."
+ *
+ * As seen above, \f$w_{\delta}\f$ controls the degree of projection of the
+ * \f$z_2\f$-base of the frustum (seen in purple) into a fourth spatial
+ * dimension \f$w\f$. The resulting grid (seen in red) is obtained by projecting
+ * back into the 3D plane at \f$w=1\f$. By default, `projective_scale_factor` is
+ * set to unity and the mapping from the logical cube to the frustum is an
+ * affine one. The user can choose to change the `projective_scale_factor`,
+ * increasing the foreshortening in the grid.
+ *
+ * The gridpoint distribution is determined by using \f$\zeta\f$ to linearly
+ * interpolate in the homogeneous coordinates (seen in purple)
+ * \f$(\tilde{x},\tilde{y},\tilde{z},w) =
+ * (\vec{x}(\xi,\eta,-1),1)\f$ to
+ * \f$(w_{\delta}\vec{x}(\xi,\eta,+1),w_{\delta})\f$, and
+ * recovering the Cartesian coordinates (seen in red) by dividing
+ * \f$\tilde{x}, \tilde{y}, \tilde{z}\f$ by \f$w\f$.
+ *
+ * The equation for the 4D frustum in homogeneous coordinates then is:
+ * \f[\begin{bmatrix}\tilde{x}\\\tilde{y}\\\tilde{z}\\w\\\end{bmatrix}=
+ * \frac{1-\zeta}{2}\begin{bmatrix}x_1\\y_1\\z_1\\1\\\end{bmatrix}+
+ * \frac{1+\zeta}{2}\begin{bmatrix}x_2w_{\delta}\\y_2w_{\delta}\\
+ * z_2w_{\delta}\\w_{\delta}\\\end{bmatrix}\f]
+ *
+ * The equation for the projected 3D frustum in Cartesian coordinates is
+ * obtained by dividing by w:
+ * \f[\vec{x}(\zeta) = \frac{1}{w(\zeta)}
+ * \begin{bmatrix}
+ * \tilde{x}(\zeta)\\
+ * \tilde{y}(\zeta)\\
+ * \tilde{z}(\zeta)\\
+ * \end{bmatrix}\f]
+ *
+ * We define an auxiliary variable `projective_zeta` \f$ = \mathrm{Z}\f$ such
+ * that the Cartesian coordinates are linear in \f$\mathrm{Z}\f$, and such that
+ * \f$\mathrm{Z}\f$ reduces to \f$\zeta\f$ when \f$w_{\delta}\ = 1\f$.
+ * \f$\mathrm{Z}\f$ is then rationally interpolated by \f$\zeta\f$.
+ * Defining the auxiliary variables \f$w_{\pm} := w_{\delta}\pm 1\f$,
+ * \f$\mathrm{Z}(\zeta)\f$ is given by:
+ * \f[\mathrm{Z} = \frac{w_- + \zeta w_+}
+ * {w_+ + \zeta w_-}\f]
+ *
+ * with derivative:
+ * \f[\mathrm{Z}' = \frac{\partial\mathrm{Z}}{\partial \zeta} =
+ * \frac{w_+^2 - w_-^2}{(w_+ + \zeta w_-)^2}\f]
+ *
+ * The full map is then given by:
+ * \f[\vec{x}(\xi,\eta,\zeta) = \begin{bmatrix}
+ * \Sigma^x + \Delta^x_{\xi}\Xi + (\Delta^x_{\zeta} +
+ * \Delta^x_{\xi\zeta}\Xi)\mathrm{Z}\\
+ * \Sigma^y + \Delta^y_{\eta}\mathrm{H} + (\Delta^y_{\zeta} +
+ * \Delta^y_{\eta\zeta}\mathrm{H})\mathrm{Z}\\
+ * \Sigma^z + \Delta^z\mathrm{Z}\\
+ * \end{bmatrix}\f]
+ *
+ * With Jacobian: \f[J =
+ * \begin{bmatrix}
+ * (\Delta^x_{\xi} + \Delta^x_{\xi\zeta}\mathrm{Z})\Xi' &
+ * 0 & (\Delta^x_{\zeta}+ \Delta^x_{\xi\zeta}\Xi)\mathrm{Z}' \\
+ * 0 & (\Delta^y_{\eta} + \Delta^y_{\eta\zeta}\mathrm{Z})\mathrm{H}' &
+ * (\Delta^y_{\zeta} + \Delta^y_{\eta\zeta}\mathrm{H})\mathrm{Z}'\\
+ * 0 & 0 & \Delta^z\mathrm{Z}' \\
+ * \end{bmatrix}
+ * \f]
+ *
+ * ### Convenient values
```

Comment:
Convenient values of w_delta (and possible replacing convenient -> Values of interest)